### PR TITLE
Fallback to known font dir paths if none were loaded via fontconfig

### PR DESCRIPTION
**Pull request by MoSal**
In cases where parsing fontconfig conf files may fail, or no font dirs
 would be loaded for some other reason, it could be useful to fallback
 to the non-fontconfig mechanism of trying to load from some fixed
 known font dir paths.

 A case where a distribution botched fontconfig conf files with an
 upgrade was seen in pop-os/cosmic-term#323. And while Nix users in
 particular may not have benefited from this change. I think
 the scenario is plausible enough to occur via distribution or user
 error.

 The benefit of things probably working, at least partially, with and
 without fontconfig's help is a big plus IMHO.

### CODE REVIEW

**Comment by RazrFalcon** [comment]
Honestly, I would rather remove fontconfig support altogether. People should stop using `fontdb` on Linux.
All other OSes work just fine and 99% of the issues come from Linux. Honestly, I just don't care at this point.

File: src/lib.rs [hunk 1]
```diff
@@ -462,31 +462,42 @@ impl Database {
         {
             #[cfg(feature = "fontconfig")]
             {
-                self.load_fontconfig();
+                if !self.load_fontconfig() {
+                    log::warn!("Failed to load font dirs via fontconfig.");
+                    log::warn!("Fallback to loading from known font dir paths.");
```
Comment (RazrFalcon) [inline src/lib.rs:467]: Remove this line. One warning line should be enough.

Comment (MoSal) [reply src/lib.rs:466]: Kept the second line.

**Comment by RazrFalcon** [comment]
Fails to build.

**Comment by MoSal** [comment]
> Fails to build.

Apologies. My bad.

**Comment by RazrFalcon** [comment]
Thanks!

**Comment by MoSal** [comment]
A version bump would be appreciated.

**Comment by RazrFalcon** [comment]
Done.

**Pull request merged**